DepotDownloader/ContentDownloader.cs

```diff
@@ -94,12 +94,13 @@ static bool TestIsFileIncluded( string filename )
             if ( !Config.UsingFileList )
                 return true;
 
-            foreach ( string fileListEntry in Config.FilesToDownload )
+            filename = filename.Replace( '\\', '/' );
+            
+            if ( Config.FilesToDownload.Contains( filename.ToLowerInvariant() ) )

```

Comment:
I’d pass a StringComparer to the HashSet’s constructor instead.

Reply:
I thought about it, doesn't that defeat the purpose of using a hashing set?